### PR TITLE
Feature/get args

### PR DESCRIPTION
**Pull request by Geam**

### CODE REVIEW

**Comment by afaucher17** [comment]
Test for github integration on slack

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,11 @@
...
 # Debug files
 *.dSYM/
+
+# CMake unwanted files
+CMakeCache.txt
+CMakeFiles
+*.cmake
+
+# program name
+serveur
```
Comment (jability) [inline .gitignore]: We also want to include our test binary '_test_zappy_server_'

Comment (jability) [inline .gitignore:40]: We also want to exclude our test binary '_test_zappy_server_'

File: src/check_arguments.c [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	if (args->height < MIN_MAP || args->height > MAX_MAP)
+		error += check_error_int("Height", args->height);
+	if (args->nb_clients < MIN_CLIENT || args->nb_clients > MAX_FD - 4)
+		error += check_error_int("Number of client", args->nb_clients);
+	if (args->tick < MIN_TICK || args->tick > MAX_TICK)
+		error += check_error_int("Tick", args->tick);
+	if (args->nb_team > (u_int)args->nb_clients / 6)
+	{
+		++error;
+		dprintf(2, "Too much team registered (%d) compare to number of "
```
Comment (jability) [inline src/check_arguments.c]: ~~Too much team registered compare to number~~ => "Too many teams registered compared to the number"

**Comment by jability** [comment]
- [x] rm_teams function is untested

**Comment by Geam** [comment]
I've correct everything

**Comment by jability** [comment]
Thanks, merging into develop

**Pull request merged**